[4.0] Make Batch button to respect "listCheck" option

Fedik opened the pull request:
Pull Request for Issue #21109 .

### Summary of Changes
Now the Batch button is grayed without selected Article

Fedik committed:
Make Batch button to respect "listCheck" option

dgrammatiko commented on layouts/joomla/toolbar/popup.php:
```diff
@@ -34,22 +34,19 @@
-	$modalAttrs['onclick'] = <<<JS
-if (document.adminForm.boxchecked.value==0){ $alert } else { jQuery( '#$selector' ).modal('show'); return true; }
-JS;
+	$modalAttrs['onclick'] = 'jQuery(\'#' . $selector . '\' ).modal(\'show\');';
```
How about `document.getElementById("' . $selector . '").open();`

Fedik replied:
I have ignored this part, I am already got lost in all the modal changes :smile: 
Thanks for note, I will change it.

Fedik committed:
Remove onclick, data-toggle/data-target is enough

Fedik committed:
Sync. Merge branch '4.0-dev' into toolbar-batch-ce

 Conflicts:
	media/system/webcomponents/js/joomla-toolbar-button-es5.js
	media/system/webcomponents/js/joomla-toolbar-button-es5.min.js
	media/system/webcomponents/js/joomla-toolbar-button.js
	media/system/webcomponents/js/joomla-toolbar-button.min.js

Fedik commented:
conflict fixed, 
note about testing:
after apply the patch you have to run `npm install`, to see a result

franz-wohlkoenig commented:
@Fedik thanks for Info " after apply the patch you have to run npm install, to see a result".

brianteeman commented:
I have tested this item :white_check_mark: successfully on c4bb28aa1256364e58d36645441049e01736d9fe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21284">issues.joomla.org/tracker/joomla-cms/21284</a>.</sub>

chmst commented:
I have tested this item :white_check_mark: successfully on c4bb28aa1256364e58d36645441049e01736d9fe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21284">issues.joomla.org/tracker/joomla-cms/21284</a>.</sub>

franz-wohlkoenig commented:
Ready to Commit after two successful tests.

laoneo commented:
Thanks!
